platform/linux-generic/odp_traffic_mngr.c

```diff
@@ -48,6 +48,9 @@ static const pkt_desc_t EMPTY_PKT_DESC = { .word = 0 };
 #define MAX_PRIORITIES ODP_TM_MAX_PRIORITIES
 #define NUM_SHAPER_COLORS ODP_NUM_SHAPER_COLORS
 
+#define TM_MIN_SHAPER_BW  1
+#define TM_MAX_SHAPER_BW  12500000UL
```

Comment:
There should be a comment about the units of these, especially if the unit is non-obvious. Now these are in the units of 1000 bytes/s but I am not sure why that is better than bits/s which is the units of the corresponding capabilities and parameters. I think it would be clearer to have these in bits/s (e.g. #define TM_MAX_SHAPER_BW (100ULL * 1000 * 1000 * 1000).

Reply:
Ack, I just moved the macros from ODP_TM_MIN_SHAPER_BW to this TM_MIN_SHAPER_BW and the unit of ODP_TM_MIN_SHAPER_BW was in 1000 bytes/sec.

I'll fix the unit to bits/sec